server/src/models/user.model.js

```diff
@@ -0,0 +1,39 @@
+require("dotenv").config();
+const { DataTypes } = require("sequelize");
+const { sequelize } = require("../configs/sequelize");
+
+const User = sequelize.define(process.env.USERS_TABLE, {
```

Comment:
Yes, since I didn't want to remove the original Users table, I created a second users table in the DEV environment named `users-dev`, so this allows us more flexibility with being able to quickly change the database tables. In the future, I can hardcode it, or just get rid of the original `Users` table and create a new one, then hardcode it